css/styles.css

```diff
@@ -0,0 +1,452 @@
+/* Styles Page */
+body {
+  background-color: #f4f5f6;
+  font-family: 'Roboto Condensed', sans-serif;
+  margin: 0 auto;
+  max-width: 1300px;
+}
+
+h1 {
+  font-family: 'Kadwa', serif;
+  letter-spacing: -3px;
+  margin: 0;
+  padding: 0;
+}
+
+h3 {
+  color: black;
+  font-size: 24px;
+  font-weight: bold;
+}
+
+.red-hover:hover {
+  text-decoration: underline red;
+}
+
+.text-danger {color: #f72210 !important;}
```

Comment:
`HINT`

Consider using a different class name from the bootstrap class name, by doing that you won't need to use `!important` to increase the CSS specificity.

For example, if you used `red-text`, there won't be a need to use `!important`.

Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`. Learn more [here](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#The_!important_exception)

Reply:
Thank you very much, that's was a great hint! I struggled for a while thinking of ways to remove those !important from my code and it never occurred to me that it was because I was using the same class name as bootstrap.